src/clients/vendor.rb

```diff
@@ -140,12 +144,12 @@ def main
             0
           )
           @vendordir = "/UnitedLinux/"
```

Comment:
UnitedLinux? That's dead for very long time... I'd get rid of this zombie once and for all. (out of scope, but if you have some time...)